### DISCUSSION TITLE
Remote host connection problem

### DISCUSSION THREAD

**Discussion by Warplex1** [rustdesk/rustdesk#4970] [Q&A]
Hello everyone,

I have a problem when I try to get my hands on a machine on a network other than my machine I have the following message displayed "(os error 10054): An existing connection was forcibly closed by the remote host"
For I did the port redirection on my internet box and on my two posts the rustdesk status is on "Ready"
Can you please help me?

**Comment by 21pages** [comment]
What version is the local rustdesk?
If the version is greater than 1.2.0, it should automatically reconnect, dit it reconnect ?
Attempt to connect with relay using id\r.
